Dalamud/Utility/TaskExtensions.cs

```diff
@@ -63,6 +63,21 @@ public static T GetResultSafely<T>(this Task<T> task)
 #pragma warning restore RS0030
     }
 
+    /// <summary>Ignores any exceptions thrown from the task.</summary>
```

Comment:
Might be good to note that this awaits the task and isn't a "modifier", looks good otherwise!

Reply:
It effectively is a modifier, as it gets and discards the exception raised if any. Caller can optionally either await for the task passed as the parameter, or the returned task. It should be taken as an another version of:
```cs
Task SuppressException(Task task) => task.ContinueWith(r => _ = r.Exception);
```

Reply:
Actually I think I get what you mean there; gonna add more docs there